### DISCUSSION TITLE
get name_column in filter type "Time column" use jinja

### DISCUSSION THREAD

**Discussion by MaximDown** [apache/superset#26223] [Q&A / Help] [answered]
Hello everyone! There are 2 filters on the dashboard.
1) The drop-down use in filter type "Time column"
2) Selected date and time. There is a virtual dataset in which several columns with dates.
I use jinja, to insert the values from the filter 2.
```{{ from_dttm }}``` and ```{{ to_dttm }}```
The problem is that I can only speed up on a certain column in the dataset. How can I make it look like this?
```
{{ name_column_in_filter1 }} >= {{ from_dttm }} and {{ name_column_in_filter1 }} < {{ to_dttm }}
```

**Comment by chaucv** [comment, marked as answer]
you can using

```
{% if(time_column == 'name_column_in_filter1') %}
            			{{from_dttm}} {{to_dttm}}
            		 {%- endif -%}
```
